hikari/api/event_manager.py

```diff
@@ -356,6 +356,7 @@ async def on_message(event):
         Wait_for: `hikari.api.event_manager.EventManager.wait_for`
         """
 
+    # FIXME: Breaking change here, make sure to document
```

Comment:
This won't pass linting until this is fixed

Reply:
As stated in the comment of this PR, intentional.

Will remove and fix when opening a pr to merge into master

Reply:
Why not just add the changelog entry now and rename it when you open the pr? The pipelines do fail until you have changelog entries.

BTW you could've communicated that better, I usually don't read comments after they've been marked as resolved since you know its hidden and I'd just assumed you wanted to add the changelog entry before merging this since you know if pipelines fail my assumption is that a pr isn't read yet

Reply:
Sorry, should have specified better. Will add the fragment changelog later and, if it's good to merge, do so and open the big pr